src/libraries/System.Text.RegularExpressions/tests/UnitTests/Resources/Strings.resx

```diff
@@ -0,0 +1,311 @@
+            : using a System.ComponentModel.TypeConverter
+            : and then encoded with base64 encoding.
+    -->
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+    <xsd:import namespace="http://www.w3.org/XML/1998/namespace" />
+    <xsd:element name="root" msdata:IsDataSet="true">
+      <xsd:complexType>
+        <xsd:choice maxOccurs="unbounded">
+          <xsd:element name="metadata">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" />
+              </xsd:sequence>
+              <xsd:attribute name="name" use="required" type="xsd:string" />
+              <xsd:attribute name="type" type="xsd:string" />
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="InvalidRegexGeneratorAttributeTitle" xml:space="preserve">
```

Comment:
is it not possible to reference existing Strings.resx's, using LogicalName or whatever it is as appropriate to ensure they are embedded correctly? That would avoid creating a new file that will need maintaining/updating.

Reply:
Tell me how and I'll do it :-)

Reply:
Include with `<EmbeddedResource>` (and `Link` I guess) in the project as you would normally. As you know, the build process then picks a name to use for the resource stream, and it probably won't be the one the existing code expects, so add `ManifestResourceName` (or possibly instead `LogicalName` if that doesn't work - they are slightly different and I can't remember how) to force the resource stream to have the name you want. (Presumably you want the same one as in the product DLL, as visible in ILSPY)

examples
https://github.com/danmoseley/runtime/blob/edfa2211c5c97d703fcad1f6f023b05f590b67f9/src/libraries/System.Speech/src/System.Speech.csproj#L233
https://github.com/danmoseley/runtime/blob/edfa2211c5c97d703fcad1f6f023b05f590b67f9/src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj#L299

You can use ILSPY on these two dll's to see the effect this metadata has on the resource stream name.

Reply:
To be clear, you don't need both. My vague memory is that LogicalName is a hammer ('exactly this')

Reply:
Also, as you know, you don't need to write like this
```xml
    <EmbeddedResource Include="Resources\Strings.resx">
      <ManifestResourceName>...</ManifestResourceName>
..
```
but it can be written like this
```xml
    <EmbeddedResource Include="Resources\Strings.resx" Link="..." ... ManifestResourceName="..." .../>

Reply:
That doesn't work as we have specialized infra around when to inject additional SR helpers.

But after looking into it, it appears just adding a `<StringResourcesPath>...</StringResourcesPath>` does the right things.